### PR TITLE
Add extra debug information to logging

### PR DESCRIPTION
**Pull request by nosami**

### CODE REVIEW

File: main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs [hunk 1]
```diff
@@ -238,11 +239,25 @@ module AsyncChoiceCE =
...
-    let logInfo format = log LoggingService.LogInfo format
-    let logWarning format = log LoggingService.LogWarning format
+
+#if DEBUG
+    let stopWatch = Stopwatch.StartNew()
+#endif
+
+    let inline private logWithThread f format =
+
+#if DEBUG
```
Comment (Therzok) [inline main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs]: Can't we add this in logging service?

Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs]: It could be added there if it's wanted.

Comment (nosami) [reply main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs]: The stopwatch elapsed time is misleading if there are multiple threads logging concurrently - I should just remove this part

Comment (Therzok) [reply main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs]: Yeah, we should have timings in general, not just the F# addin. /Cc @slluis

**Comment by slluis** [comment]
@nosami if you still need this, please resolve conflicts and merge.

**Comment by nosami** [comment]
I removed the stopwatch code and rebased this twice now. The first time, there was an unrelated test failure, this time the Unit Tests build seems to be stuck in `Pending`

**Pull request merged**